www/index.html

```diff
@@ -487,10 +491,19 @@ <h2 data-i18n="configure-title">Configuration</h2>
                             <br /> Scanning for archives... Please wait <img src="img/spinner.gif" alt="Please wait..." />
                         </div>
                         <div id="chooseArchiveFromLocalStorage" style="display: none;">
-                            <br /> Please select the archive you want to use : <select id="archiveList"
-                                class="form-control"></select>
-                            <br /><button type="button" class="btn btn-light" id="btnRescanDeviceStorage">Rescan</button>
-                            Rescans your SD Cards and internal memory
+                            <div id="filesSelectionInfoAndCount">
+                                <p id="numberOfFilesCount" style="display: none;">0</p>
+                                <p id="fileCountDisplay" style="display: none;" data-i18n="configure-select-file-numbers">
+                                    archives found in selected location.&nbsp;
```

Comment:
You need to add the parentheses, `archive(s)` here too, as this can act as fallback if the browser can't load the translation strings.